linera-views/src/value_splitting.rs

```diff
@@ -51,6 +51,11 @@ where
     type Keys = Vec<Vec<u8>>;
     type KeyValues = Vec<(Vec<u8>, Vec<u8>)>;
 
+    fn max_key_size(&self) -> usize {
+        // We decrease by 4 because of the constraint
```

Comment:
This sentence doesn't help much. Remove it?

Reply:
Ok.